classes/functions.php

```diff
@@ -58,7 +58,7 @@ public function issueLookup($comic_id) {
 			die ( "Connection failed: " );
 		}
 
-		$sql = "SELECT comics.wiki_id, comics.series_id, comics.comic_id, series.series_name, series.series_vol, comics.issue_number, comics.release_date, comics.story_name, comics.cover_image, comics.plot, comics.original_purchase FROM comics LEFT JOIN series ON comics.series_id=series.series_id WHERE comics.comic_id = $comic_id";
+		$sql = "SELECT * FROM comics LEFT JOIN series ON comics.series_id=series.series_id WHERE comics.comic_id = $comic_id";
```

Comment:
Seemed excessive to write out all those fields rather than just doing a wildcard selector *. LMK if there was a specific reason and I'll restore it.